src/languages/json-language.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @filedescription Functions to fix up rules to provide missing methods on the `context` object.
+ * @author Nicholas C. Zakas
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import { parse } from "@humanwhocodes/momoa";
+import { JSONSourceCode } from "./json-source-code.js";
+
+//-----------------------------------------------------------------------------
+// Types
+//-----------------------------------------------------------------------------
+
+/** @typedef {import("@humanwhocodes/momoa").DocumentNode} DocumentNode */
+/** @typedef {import("@eslint/core").Language} Language */
+/** @typedef {import("@eslint/core").File} File */
+/** @typedef {import("@eslint/core").ParseResult} ParseResult */
+/** @typedef {import("@eslint/core").SyntaxElement} SyntaxElement */
+
+//-----------------------------------------------------------------------------
+// Exports
+//-----------------------------------------------------------------------------
+
+/**
+ * JSON Language Object
+ * @implements {Language}
+ */
+export class JSONLanguage {
```

Comment:
Missing `visitorKeys` property?

Reply:
`visitorKeys` is optional, so leaving off for now.

Reply:
Okay, but it would be good to add this at some later point to not rely on fallbacks.

Reply:
Agreed. As I said in the PR, just want to get a base checked in rather than trying to get everything perfect in this first PR.